playbooks/viaq-store.yml

```diff
@@ -155,108 +147,6 @@
     - name: Wait till done
       shell: oc rollout status -w $( oc get -n logging dc -l component=es -o name )
 
-
-
-- name: Post-Install
-  hosts: metrics-store
-  tasks:
-    - name: change project
-      shell: oc project logging
-    - name: Getting CA cert
-      shell: oc get secret logging-fluentd --template='{{ '{{' }}index .data "ca"{{ '}}' }}'  | base64 -d > es-ca.crt
-    - name: Getting client CA cert
-      shell: oc get secret logging-fluentd --template='{{ '{{' }}index .data "cert"{{ '}}' }}'  | base64 -d > es-client-ca.crt
-    - name: Getting the shared key
-      shell: oc get secret logging-fluentd --template='{{ '{{' }}index .data "key"{{ '}}' }}' | base64 -d > es-shared-key
-    - name: get ovirt_env_uuid_metrics
-      shell: oc get project ovirt-metrics-test-engine -o jsonpath='{.metadata.uid}' > ovirt_env_uuid_metrics
-    - name: get ovirt_env_uuid_logs
-      shell: oc get project ovirt-logs-test-engine -o jsonpath='{.metadata.uid}' > ovirt_env_uuid_logs
-    - name: Do magic with admin
-      shell:  oc login --username=admin --password=admin
-    - name: Do magic with sysytemadmin
-      shell: oc login --username=system:admin
-    - name: Do magic with cluster-admin
-      shell: oadm policy add-cluster-role-to-user cluster-admin admin
-    - name: Fetch the created files
-      fetch:
-        src: "{{ item }}"
-        dest: /tmp/
-        flat: yes
-      with_items:
-        - "/root/es-ca.crt"
-        - "/root/es-client-ca.crt"
-        - "/root/es-shared-key"
-        - "/root/ovirt_env_uuid_metrics"
-        - "/root/ovirt_env_uuid_logs"
-
-- name: Copy metrics store configuration to engine machine
-  hosts: engine
-  tasks:
-    - name: store certificate on engine machine
-      copy:
-        src: "/tmp/{{ item }}"
-        dest: "/etc/ovirt-engine-metrics/{{ item }}"
-      with_items:
-        - "es-ca.crt"
-        - "es-client-ca.crt"
-        - "es-shared-key"
-        - "ovirt_env_uuid_metrics"
-        - "ovirt_env_uuid_logs"
-
-
-- name: Configure config.yml on engine machine
-  hosts: engine
-  vars:
-     ovirt_env_uuid_metrics: "{{ lookup('file', '/tmp/ovirt_env_uuid_metrics') }}"
-     ovirt_env_uuid_logs: "{{ lookup('file', '/tmp/ovirt_env_uuid_logs') }}"
-  tasks:
-    - name: ensure config.yml exists
-      file:
-        path: /etc/ovirt-engine-metrics/config.yml
-        state: touch
-    - name: Display all variables/facts known for a host
-      debug:
-        msg: "Debug - {{ hostvars[groups['metrics-store'][0]] }}"
-    - name: Save metrics store machine fqdn
-      set_fact:
-        metrics_store_fqdn: "{{ hostvars[groups['metrics-store'][0]]['ansible_fqdn'] }}"
-    - name: add ovirt_env_name parameters to config.yml file
-      lineinfile:
```

Comment:
I would still like to modify the config.yml in the script, and not manually

Reply:
Added back the relevant tasks.